### PR TITLE
feat: adds <Card /> components

### LINKED ISSUE #8: create <Card /> component
https://pattern-library.dequelabs.com/composites/tiles

### PR DESCRIPTION
**Pull request by schne324**
resolves #8

### CODE REVIEW

File: types.d.ts [hunk 1]
```diff
@@ -262,3 +262,27 @@ export class AriaIsolate {
 }
 
 export const focusableSelector: string;
+
+interface CardProps {
```
Comment (stephenmathieson) [inline types.d.ts]: Since these components take any arbitrary props, it'd be better to just give them `React.HTMLAttributes<HTMLDivElement>`.

Comment (stephenmathieson) [reply types.d.ts]: No worries!

File: types.d.ts [hunk 2]
```diff
@@ -262,3 +262,8 @@ export class AriaIsolate {
 }
 
 export const focusableSelector: string;
+
+export const Card: React.HTMLAttributes<HTMLDivElement>;
```
Comment (stephenmathieson) [inline types.d.ts]: This is still not quite right. This should be:

```ts
export const Card: React.ComponentType<React.HTMLAttributes<HTMLDivElement>>
```

This basically means `Card` is of `React.ComponentType` and takes `React.HTMLAttributes<HTMLDivElement>` as its props.

Comment (schne324) [reply types.d.ts]: got it. is there any way I can test this stuff out easily? That way I could validate that my changes "work"

Comment (stephenmathieson) [reply types.d.ts]: Yep! See the [`test/typechecks.ts` file](https://github.com/dequelabs/cauldron-react/blob/master/test/typechecks.tsx).

**Pull request merged**